File: it/expected/tests/code/java/test/Entry.java
package test;

import com.fasterxml.jackson.annotation.JsonCreator;

public class Entry {
  @JsonCreator
  public Entry() {
  }

  @Override
  public int hashCode() {
    int result = 1;
    return result;
  }

  @Override
  public boolean equals(final Object other) {
    if (other == null) {
      return false;
    }

    if (!(other instanceof Entry)) {
      return false;
    }

    @SuppressWarnings("unchecked")
    final Entry o = (Entry) other;

    return true;
  }

  @Override
  public String toString() {
    final StringBuilder b = new StringBuilder();

    b.append("Entry");
    b.append("(");
    b.append(")");

    return b.toString();
  }

  public static class Builder {
    public Entry build() {

      return new Entry();
    }
  }
}

File: it/structures/csharp/service-default/Test/Entry.cs
using Newtonsoft.Json;
using System;

namespace Test {
    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
    public class Entry {

        [JsonConstructor]
        public Entry () {}

        public override bool Equals(Object other) {
            Entry o = other as Entry;

            if (o == null) {
                return false;
            }

            return true;
        }

        public override int GetHashCode() {
            int result = 1;
            return result;
        }

        public override String ToString() {
            return "Entry()";
        }
    }
}
